source/_posts/2016-10-01-we-have-raspberry-image-now.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "We have a Raspberry Pi image now"
+description: "Release of the Rapsberry Pi Image for Home Assistant"
+date: 2016-10-01 01:00:00 -0400
+date_formatted: "October 1, 2016"
+author: Fredrik Lindqvist
+comments: true
+categories: Technology
+---
+
+## {% linkable_title We finally have a Raspberry Pi image! %}
```

Comment:
It looks like two titles when rendered. 

![title](https://cloud.githubusercontent.com/assets/116184/19015265/34d1e8d6-8801-11e6-81c6-49ea9d3a5543.png)